### PR TITLE
tree: update to 2.0.4.

### PR DESCRIPTION
**Pull request by skmpz**
#### Testing the changes
- I tested the changes in this PR: **YES**

### CODE REVIEW

File: srcpkgs/tree/template [hunk 1]
```diff
@@ -1,16 +1,16 @@
...
-version=2.0.3
+version=2.0.4
 revision=1
+wrksrc="unix-tree-${version}"
 build_style=gnu-makefile
 short_desc="Recursive directory listing program"
 maintainer="Orphaned <orphan@voidlinux.org>"
 license="GPL-2.0-or-later"
 homepage="http://mama.indstate.edu/users/ice/tree"
 changelog="http://mama.indstate.edu/users/ice/tree/changes.html"
```
Comment (classabbyamp) [inline srcpkgs/tree/template:11]:
```suggestion
homepage="https://mama.indstate.edu/users/ice/tree"
changelog="https://mama.indstate.edu/users/ice/tree/changes.html"
```

Comment (classabbyamp) [inline srcpkgs/tree/template:11]:
```suggestion
homepage="https://mama.indstate.edu/users/ice/tree"
changelog="https://mama.indstate.edu/users/ice/tree/changes.html"
```

**Comment by classabbyamp** [comment]
117a4d18fb1722543600209f9252285be14752fd

**Comment by sgn** [comment]
117a4d18fb1722543600209f9252285be14752fd

**Pull request closed without merging**